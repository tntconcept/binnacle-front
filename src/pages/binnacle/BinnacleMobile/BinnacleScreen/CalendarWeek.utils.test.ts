import {
  getDaysOfWeek,
  getNextWeek,
  getPreviousWeek
} from 'pages/binnacle/BinnacleMobile/BinnacleScreen/CalendarWeek.utils'

describe('Calendar week', () => {
  const date = new Date('2020-01-31T20:10:00.000Z')

  it('should return each day of week', function() {
    const result = getDaysOfWeek(date)
    expect(result).toMatchSnapshot()
  })

  it('should return previous week', function() {
    const result = getPreviousWeek(date)
    expect(result).toEqual(new Date('2020-01-24T20:10:00.000Z'))
  })

  it('should return next week', function() {
    const result = getNextWeek(date)
    expect(result).toEqual(new Date('2020-02-07T20:10:00.000Z'))
  })
})
