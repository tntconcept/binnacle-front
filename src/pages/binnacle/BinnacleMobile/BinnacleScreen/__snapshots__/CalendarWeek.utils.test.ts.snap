// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calendar week should return each day of week 1`] = `
Array [
  2020-01-26T23:00:00.000Z,
  2020-01-27T23:00:00.000Z,
  2020-01-28T23:00:00.000Z,
  2020-01-29T23:00:00.000Z,
  2020-01-30T23:00:00.000Z,
  2020-01-31T23:00:00.000Z,
  2020-02-01T23:00:00.000Z,
]
`;
